content/20.results.md

```diff
@@ -19,16 +19,31 @@ Because the concordance was high and the PubMed set was more complete, we used t
 
 ### Assessing Gender Diversity of Authors and Honorees
 
+Although Bioinformatics was established in 1998 and BMC Bioinformatics in 2000, the metadata for these journal papers before 2002 only have initials for first and/or middle author names.
+Therefore, without first and middle names, we do not have author gender predictions before this year.
+
+There is a slow increasing trend of the proportion of female authors over the years, arriving at just over 20% on average in 2019 (Fig. {@fig:gender_breakdown}A).
+Analyzing each journal individually, we observe very similar trend across the three selected journals, but the least increase in the estimated female proportion is found in PLOS Computational Biology (Supplementary Fig S1).
+ISCB fellows and keynote speakers appear to be more evenly split between men and women compared to the population of authors published in computational biology and bioinformatics journals (Fig. {@fig:gender_breakdown}B).
+The overall increasing female proportion trend seems to be present in each honor category, especially in the group of ISCB Fellows (Fig. {@fig:gender_breakdown}C).
+It is more difficult to make inference from the group of PSB speakers because the conference only has 1-2 keynotes each year (Fig. {@fig:gender_breakdown}C).
+
+[Should we include C here or put in supplement?]
```

Comment:
What do you think? I think it's notable that PSB 2020 is the first major conference with all female keynotes, but PSB also has more of a track record of all female keynotes (b/c so few of them). I trust your judgement on what you think the way to go is.

Reply:
I think it's interesting to look at the number for each conference. I'm happy keeping it in. Your earlier comment touched on the possibility of all female/male keynotes because there are max two keynotes, but we can reiterate this fact if you think it's noteworthy!